src/main/java/org/sonar/plugins/checkstyle/metadata/CheckstyleMetadata.java

```diff
@@ -239,30 +304,38 @@ public static boolean isSetter(Method method) {
      * It converts the name received from ModuleDetails to Sonar rule name format
      * e.g. RightCurlyCheck -> Right Curly Check
      *
-     * @param name the name fetched from ModuleDetails
+     * @param checkName the name fetched from ModuleDetails
      * @return modifiedName
      */
-    private static String convertName(String name) {
+    private static String convertName(String checkName) {
```

Comment:
make this method public to be reused in test code.
no need to copy it.

Reply:
Done in new reorganized PR https://github.com/checkstyle/sonar-checkstyle/pull/306